Client/src/index.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
```

Comment:
The Udemy course was doing a bad practice. A React `index.js` file should be an entry file with minimal lines of code. Imports should be used.